tests/communication/test_integration.py

```diff
@@ -40,7 +41,9 @@ def test_semeio_script_integration(tmpdir):
     assert list(range(10)) == reported_data
 
     # Assert that logs were forwarded correctly
-    with open("storage/reports/TestWorkflowJob/log.txt") as f:
+    with open(
+        os.path.join("storage/reports/TestWorkflowJob/", SEMEIOSCRIPT_LOG_FILE)
```

Comment:
Perhaps we can store the filename in a variable? I agree with the black formatting, but at some point a variable makes the whole thing more readable...